comp/metadata/internal/util/inventory_payload.go

```diff
@@ -69,7 +69,7 @@ import (
 
 var (
 	defaultMinInterval = 1 * time.Minute
-	defaultMaxInterval = 10 * time.Minute
+	defaultMaxInterval = 1 * time.Minute
```

Comment:
Is this change intended? What is the difference between `defaultMinInterval` and `defaultMaxInterval`?